pylintrc

```diff
@@ -7,7 +7,7 @@
 # progress
 disable=too-many-lines,locally-disabled,duplicate-code,too-few-public-methods,
     bad-option-value,no-else-return,cyclic-import,too-many-public-methods,
-    bad-continuation
+    bad-continuation, raise-missing-from
```

Comment:
This one is a little special. The `raise from` form actually adds information to the traceback, which is useful and I therefore think this is one of the additions to the pylint checker that I fully agree with. Se here for an explanation of the error: https://stackoverflow.com/a/63852262/11640721 and here for a little more explanation of the `raise from` form: https://stackoverflow.com/questions/24752395/python-raise-from-usage

I don't know how many try-excepts will need to be fixed, but I can lend a hand if it is many.

Reply:
Yep. I've actually already gone ahead and fixed up the `raise from` warnings. Decided it was better once I understood it.

Might also want to specify `pylint >= 2.6` in `requirements-dev.txt`; what do you think?

Reply:
Yeah, that is probably a good idea.